src/api/task.js

```diff
@@ -0,0 +1,48 @@
+import request from './request'
+
+export default {
+  /**
+   * Get list of all tasks
+   * @return {Promise}
+   */
+  async get() {
+    const res = await request.get('tasks')
+    return await res.json()
+  },
+
+  /**
+   * Gets specifik task based on task id
+   * @param {Number} id
+   */
+  async get(id) {
+    const res = await request.get(`tasks/${id}`)
+    return await res.json()
+  },
+
+  /**
+   * Create new Task from Json body
+   * @param {Object} body
+   */
+  async create(body) {
+    const res = await request.post('tasks', body)
```

Comment:
When #33 is merge, then adopt the superstruct for body validation.

Reply:
Do we merge mine first, and then make a new PR when the superstruct works,, or should I just wait for the superstruckt?